armi/utils/pathTools.py

```diff
@@ -62,6 +62,8 @@ def copyOrWarn(fileDescription, sourcePath, destinationPath):
         runLog.debug(
             "Copied {}: {} -> {}".format(fileDescription, sourcePath, destinationPath)
         )
+    except shutil.SameFileError:
+        pass
```

Comment:
Huh... how has this not been blowing things up before? I suppose this only happens with absolute paths, but I would think we would avoid trying to copy them altogether